jenkins/common.groovy

```diff
@@ -206,18 +206,18 @@ def build(String ref, dockerContainer, String assetSource, String envName, Boole
   def drupalCred = DRUPAL_CREDENTIALS.get('vagovprod')
   def drupalMode = useCache ? '' : '--pull-drupal'
   def drupalMaxParallelRequests = 15
-  def noProxy = '--no-proxy'
+  def drupalProxy = '--no-drupal-proxy'
```

Comment:
This is confusing me as the var is drupalProxy but actually means no proxy.

Reply:
it didn't bother me since it was just a string value anyway but good idea